internal/controller/datadogagent/component/clusteragent/utils.go

```diff
@@ -53,6 +58,27 @@ func GetClusterAgentService(dda metav1.Object) *corev1.Service {
 	return service
 }
 
+func GetClusterAgentPodDisruptionBudget(dda metav1.Object) *policyv1.PodDisruptionBudget {
```

Comment:
I put it in `utils.go` because `GetClusterAgentService` is in `utils.go` and it follows a similar structure as `GetClusterAgentPodDisruptionBudget`. Should I move it to `default.go` for consistency between DCA and CCR?